corehq/ex-submodules/phonelog/utils.py

```diff
@@ -231,10 +231,11 @@ def _fill_base_template(self, log):
     def _get_user_info(self, log):
         user_subreport = _get_logs(self.xform.form_data, 'user_subreport', 'user')
         username, user_id = _get_user_info_from_log(self.domain, log)
+        #  Use the first user subreport to infer username and user id, if it's available
```

Comment:
this could be written as the following so that we don't need to repeat the conditional

```
if not user_subreport:
    return username or '', user_id or ''

# try to infer
```